20240731_schulddossier_template_data_only.sql

```diff
@@ -1,97 +0,0 @@
-TRUNCATE public.dossier CASCADE;
```

Comment:
Dit blijft helaas wel allemaal in git staan. Moeten we met Jeffrey overleggen hoe we dit echt volledig uit Git en Github krijgen zodat we de repo weer op public kunnen zetten?

Github heeft volgens mij ook intern allerlei caches en files enzo waar het nog in staat. Vandaar dat we dat (naar ik me kan herinneren) niet volledig zelf kunnen.

Reply:
Het betreft in beide documenten fictieve data, Team 7 heeft de dossiers die in de file staan zelf met de hand opgebouwd met testdata.
Het verwijderde bestand is niet het bestand wat destijds een datalek heeft veroorzaakt, maar een voorganger van het huidige bestand.

Banknamen zijn opzich geen probleem omdat dit geen persoonsgegevens zijn, dus die zouden door moeten kunnen.

Reply:
Helemaal goed! Good shit